app/controllers/groups_controller.rb

```diff
@@ -114,6 +114,13 @@ def index
 
   def populate
     @assignment = Assignment.find(params[:assignment_id])
+
+    # Since peer review assignments are shallow clones of an assignment, any
```

Comment:
It is not true that peer review assignments are shallow clones (their properties are not identical).

But anyway, this change isn't good, and should be removed. I'll comment on how you're populating the tables below.